content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -82,27 +82,184 @@ Costing method
-     both of inventory on hand and the costs accumulated from validated purchase orders.
-
-- :guilabel:`First In First Out (FIFO)`: tracks the costs of incoming and outgoing items in
-  real-time and uses the real price of the products to change the valuation. The oldest purchase
-  price is used as the cost for the next good sold until an entire lot of that product is sold. When
-  the next inventory lot moves up in the queue, an updated product cost is used based on the
-  valuation of that specific lot. This method is arguably the most accurate inventory valuation
-  method for a variety of reasons, however, it is highly sensitive to input data and human error.
+
+.. tabs::
+
+   .. tab:: Standard Price
+
+      The default costing method in Odoo. The cost of the product is manually defined on the product
+      form, and this cost is used to compute the valuation. Even if the purchase price on a purchase
+      order differs, the valuation is the cost defined on the product form.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $10
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $10
+           - 12
+           - 4 * $10
+           - $120
+         * - Deliver 10 products
+           - $10
+           - 2
+           - -10 * $10
+           - $20
+         * - Receive 2 products for $9/unit
+           - $10
+           - 4
+           - 2 * $10
+           - $40
+
+   .. tab:: Average Cost (AVCO)
+
+      Calculates the valuation of a product based on the average cost of that product, divided by
+      the total number of available stock on-hand. With this costing method, inventory valuation is
+      *dynamic*, and constantly adjusts based on the purchase price of products.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $0
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $12
+           - 12
+           - 4 * $16
+           - $144
+         * - Deliver 10 products
+           - $12
+           - 2
+           - -10 * $12
+           - $24
+         * - Receive 2 products for $6/unit
+           - $9
+           - 4
+           - 2 * $6
+           - $36
+
+      .. spoiler:: How are Unit Cost and Inventory Value calculated at each step?
```

Comment:
Spoiler formatting seems to break when inside a tab, lets pull the content out of the spoiler admonition :)

Also, the formatting on "unit cost" and "inventory value" can be removed.
```suggestion
      How are unit cost and inventory value calculated at each step?
```